example/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -447,6 +448,7 @@
 /* Begin XCBuildConfiguration section */
 		33CC10F92044A3C60003C045 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Generated.xcconfig */;
```

Comment:
We should follow the iOS model of having a Debug.xcconfig and a Release.xcconfig that are used as bases, and have them import the Generated.xcconfig. That makes things more consistent between the two platforms, and gives us flexibility in the future.

Reply:
Done